### ISSUE TITLE
doesn't work dev-server

### ISSUE THREAD

**Issue by dimorin** [phuoc-ng/csslayout#162]
When I  did 'npm run dev-server', I got message 'Failed to compile'.
What should I do? Could you advise for me?

Error is below :

ERROR in ./client/index.tsx
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in E:\csslayout\node_modules\@babel\helper-compilation-targets\package.json
    at throwExportsNotFound (internal/modules/esm/resolve.js:290:9)
    at packageExportsResolve (internal/modules/esm/resolve.js:513:3)
    at resolveExports (internal/modules/cjs/loader.js:432:36)
    at Function.Module._findPath (internal/modules/cjs/loader.js:472:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:867:27)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (E:\csslayout\node_modules\@babel\preset-env\lib\debug.js:8:33)        
    at Module._compile (internal/modules/cjs/loader.js:1063:30)

**Comment by alejandroBrb** [comment]
With node 12.20.1 I have the same issue, but with node 10.* I have a different issue 

```
ERROR in ~/csslayout/client/patterns/sticky-table-column/Details.tsx(95,73)
      TS2322: Type '({ padding: string; } & { backgroundColor: string; left: number; position: string; }) | ({ padding: string; } & { backgroundColor?: undefined; left?: undefined; position?: undefined; })' is not assignable to type 'CSSProperties'.
  Type '{ padding: string; } & { backgroundColor: string; left: number; position: string; }' is not assignable to type 'CSSProperties'.
    Types of property 'position' are incompatible.
      Type 'string' is not assignable to type 'Position'
```

**Comment by phuoc-ng** [comment]
Fixed in the #163.
